### ISSUE TITLE
Add support for the CoRE specification

### ISSUE THREAD

**Issue by nashif** [zephyriot/zep-jira14#1031]
**_Reported by Vinicius Costa Gomes:_**

The CoRE specification [RFC6690|https://tools.ietf.org/html/rfc6690] describes ways to list and discover the relationship between resources in a constrained environment.

(Imported from Jira ZEP-1137)

**Comment by nashif** [comment]
**_by Anselmo L. S. Melo:_**

https://gerrit.zephyrproject.org/r/#/c/6562/ is merged.
_Vinicius Costa Gomes_ , anything else pending on this?

**Comment by nashif** [comment]
**_by Vinicius Costa Gomes:_**

Nothing pending. Closing. Thanks.

**Comment by nashif** [comment]
**_by Vinicius Costa Gomes:_**

Got merged into the net branch in commit 282ee076cfd5760f519f69182ced0fc9bc6cc711
